pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -83,6 +83,9 @@ type GardenletConfiguration struct {
 	// by the Gardenlet in the seed clusters.
 	// +optional
 	SNI *SNI `json:"sni,omitempty"`
+	// ExposureClassHandlers is a list of optional of exposure class handlers.
+	// +optional
+	ExposureClassHandlers []ExposureClassHandler `json:"exposureClassHandlers,omitempty"`
```

Comment:
Please enhance https://github.com/gardener/gardener/blob/master/example/20-componentconfig-gardenlet.yaml and https://github.com/gardener/gardener/blob/master/charts/gardener/gardenlet/charts/runtime/templates/configmap-componentconfig.yaml

Reply:
I updated the chart. As said the examples will follow in a subsequent pr.